sdk/go/dagger/api/api.gen.go

```diff
@@ -233,6 +233,16 @@ func (r *Container) From(address ContainerAddress) *Container {
 	}
 }
 
+// This container's root filesystem. Mounts are not included.
+func (r *Container) FS() *Directory {
```

Comment:
@aluzzardi Does the codegen special-case this to yield `FS` instead of `Fs`? :thinking:

Reply:
https://github.com/dagger/dagger/blob/de0bcf63059a70137515bce5375ce7c1e561da70/codegen/generator/templates/lint_name.go#L62